### ISSUE TITLE
Cache mapping of aliases -> uuids

### ISSUE THREAD

**Issue by anavarre** [acquia/cli#296]
E.g.

```
$ time acli api:environments:database-list <SITEGROUP>.<ENV>
0m8.907s
```
Using an alias instead of a UUID makes an extra API call and it'll inherently be slow.

We could add some caching so we have a mapping of aliases -> UUID that would prevent from querying the API always. This never changes so it's safe to cache.

**Comment by danepowell** [comment]
Specifically, we should map both sitegroups to application IDs, and environments to environment IDs.

**Comment by anavarre** [comment]
Fix in #298
